package.json

```diff
@@ -92,6 +92,7 @@
     "devDependencies": {
         "@babel/eslint-parser": "7.17.0",
         "@babel/plugin-transform-runtime": "7.8.3",
+        "@babel/helper-create-regexp-features-plugin": "7.22.5",
```

Comment:
not sure what's the need for this. I don't see bootstrap issues on renative when testing locally